paasta_tools/check_flink_services_health.py

```diff
@@ -70,7 +134,22 @@ def check_flink_service_health(
     num_healthy_jobmanagers = healthy_flink_containers_cnt(si_pods, "jobmanager")
     num_healthy_taskmanagers = healthy_flink_containers_cnt(si_pods, "taskmanager")
 
-    # TBD: check cnt according to Flink
+    strerror = None
+    reported_taskmanagers = None
+    try:
+        overview = flink_tools.get_flink_jobmanager_overview(
+            instance_config.service, instance_config.instance, instance_config.cluster
+        )
+        reported_taskmanagers = overview.get("taskmanagers", 0)
+    except ValueError as e:
+        strerror = str(e)
+
+    send_event_if_not_enough_taskmanagers(
+        instance_config=instance_config,
+        expected_count=taskmanagers_expected_cnt,
+        num_reported=reported_taskmanagers,
+        strerror=strerror,
```

Comment:
It's actually surprising that `mypy` says nothing here.
`strerror` and `reported_taskmanagers` can be `None`, but are passed into a function expecting `str` and `int` types accordingly.

Reply:
Thank you, I added `Optional` here. Indeed it is surprising that mypy missed it.